src/app.js

```diff
@@ -1,18 +1,30 @@
 import {run} from '@cycle/core';
 import CycleDOM from '@cycle/dom';
-import {Observable} from 'rx'
-import Todos from './components/Todos/index';
-// import CustomDrivers from './drivers';
+import {Observable} from 'rx';
 import storageDriver from '@cycle/storage';
-import todos from './components/todos/index';
+// THE MAIN FUNCTION
+// This is the todo list component.
+import Todos from './components/Todos/index';
 
 const main = Todos;
 
+// THE ENTRY POINT
+// This is where the whole story starts.
+// `run` receives a main function and an object with
+// the source streams.
```

Comment:
"and an object with the source streams"
That's wrong, it's an object with the drivers (each driver is a function). Drivers may generate source streams, but not necessarily. Some drivers just read sinks and consume them, generating no sources at all.